src/OpenColorIO/GpuShaderClassWrapper.cpp

```diff
@@ -7,6 +7,8 @@
 
 #include "GpuShaderClassWrapper.h"
 
+#include <cctype>
```

Comment:
This include is already present on line 4.

Reply:
Whoops! Was not there in RB-2.1. Will get rid of the extra.